src/main/java/org/example/Commands.java

```diff
@@ -180,184 +148,81 @@ public void onMessageReceived(MessageReceivedEvent event){
+                        ArrayList<String> userInventory = server.getUserInventory(event.getMember().getId());
+
+                        if(userBadges.contains(requestedBadge) || userInventory.contains(searchQuery)) {
+                            event.getChannel().sendMessage(errorEmote + "You already have this badge either displayed or in inventory.").queue();
+                            break;
+                        }
+
+                        //transaction is done, and adds badge to user inventory before equipping the badge
+                        updateCredits(event,Integer.valueOf(badgeList.get(searchQuery).get(4)),false);
+                        server.addBadgeToInventory(String.valueOf(event.getMember().getIdLong()),searchQuery, requestedBadge);
+                        event.getChannel().sendMessage("Transaction complete... your new badge has been added to your inventory. " + boxEmote).queue();
+
+                        if(userBadges.size() >= 4){ //checking if you have an available badge slot.
+                            String userBadgeSlots = "《 " + userBadges.get(0) + " | " + userBadges.get(1) + " | " + userBadges.get(2) + " | "+ userBadges.get(3) + " 》";
+                            msgEmbed.setColor(Color.WHITE);
+                            msgEmbed.setTitle(errorEmote + "You currently have the maximum amount of badges that can be equipped at a time.");
+                            msgEmbed.setDescription(replaceBadgeMessage);
+                            msgEmbed.addField("Your Current Badge Slots:",userBadgeSlots, false);
+                            event.getChannel().sendMessageEmbeds(msgEmbed.build()).queue();
+                            msgEmbed.clear();
+                            break;
+                        }
+                        server.equipBadge(event.getMember().getId(), requestedBadge);
+                        event.getChannel().sendMessage("Your new badge has been added to your credit card, enjoy!!! " + pepeDS).queue();
+                    }
+                    break;
+                case "replacebadge":
+                    creditCardObject.replaceBadge(event, badgeList, args[1], args[2], server);
+                    break;
+                case "equipbadge":
+                    creditCardObject.equipBadge(event, badgeList, args[1], server);
+                    break;
+                case "unequipbadge":
+                    creditCardObject.unequipBadge(event, badgeList, args[1], server);
+                    break;
+                case "clearbadges":
+                    creditCardObject.clearBadges(event, server);
+                    break;
+                case "addbadge":
+                    if(addBadgeObject.addNewBadge(event,server, args[1], args[2], args[3], args[4], Arrays.copyOfRange(args, 5, args.length)));
+                    break;
+                case "inventory":
+                    inventoryObject.printInventoryEmbed(event,server,event.getMember().getId());
```

Comment:
Missing a break case here causes inventory to wipe after inventory command is called.